sonic_package_manager/service_creator/creator.py

```diff
@@ -231,6 +232,17 @@ def generate_container_mgmt(self, package: Package):
 
         run_opt.append('-t')
 
+        if not is_asic_service:
+            if container_spec['network']:
+                docker_network_type = container_spec['network']
+                run_opt.append(f'--net={docker_network_type}')
+        else:
+            if container_spec['network']:
+                raise ServiceCreatorError(f"Invalid Configuration, asic-service must not contain network type")
```

Comment:
>not contain network type [](http://example.com/codeflow?start=85&length=24)

empty value means default value. It is not proper to enforce empty value only.

Reply:
But ASIC services should not have network specified. This comment was given by the NVIDIA SONiC team. I am missing something?

Reply:
what do you suggest? to enforce the `none` as the docker_network_type like this
`run_opt.append(f'--net={docker_network_type}')`

Reply:
Sorry, let me clarify:
1. container_spec['network'] should have default value `bridge`. ie. if it is None, treat it as `bridge`.
2. I do not propose to append run_opt, always respect user config
3. if is_asic_service and container_spec['network'] != none, you can raise an exception.

Reply:
I see, but this is what I do in this if and else that i added. 
for point 3, if the container_spec['network'] is None I raise exception, am I missing something?

Reply:
Are you suggesting to do explicitly `container_spec['network'] != 'none'`?

Reply:
yes